firebase.json

```diff
@@ -1,16 +1,35 @@
+// 
 {
-  "hosting": {
-    "public": "dist",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
-    "rewrites": [
-      {
-        "source": "**",
-        "destination": "/index.html"
-      }
-    ]
-  }
+  "hosting": [
+    {
+      "site": "leeway-staging-f2db3", // This could be "react-app" if that's what you've used before
+      "public": "dist", // The directory containing your React app build
+      "ignore": [
+        "firebase.json",
+        "**/.*",
+        "**/node_modules/**"
+      ],
+      "rewrites": [
+        {
+          "source": "**",
+          "destination": "/index.html"
+        }
+      ]
+    },
+    {
+      "site": "storybook-2c67d",
+      "public": "storybook-static", // The directory containing your Storybook build
```

Comment:
The "storybook-static" folder is the result of the build process and contains compiled files that are easily regenerated. Generally, compiled or generated files, temporary files, and user-specific configuration files should not be tracked in a Git repository. This is because they can easily be recreated from the source code or project files, and tracking them can bloat the repository size and lead to versioning issues. Instead, these files should be added to the repository's ".gitignore" file to exclude them from version control.

Reply:
yes i will do that